### DISCUSSION TITLE
Mutlipe Slaves over SPI

### DISCUSSION THREAD

**Discussion by aeropagz** [UncleRus/esp-idf-lib#507] [Q&A] [answered]
How do I use multiple Slave over SPI. I am using 2 mcp23s17. 
As soon as I add a second device I don't get any values from the first one. I assigned both a CS Pin. 
Does anyone know some code examples for spi usage with 2 slaves on the same spi host?

**Comment by trombik** [comment]
we cannot comment on your issue unless you provide further clarifications, like the wiring, and the code. note that here is not a place to learn SPI. Analog Devices has [a good introduction to SPI](https://www.analog.com/en/analog-dialogue/articles/introduction-to-spi-interface.html). Use an oscilloscope or a logic analyzer, and see what is happening on the wire.

**Reply by aeropagz** [reply, marked as answer]
Turns out it was a hardware issue. Is working now
